examples/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+services:
+  nexus_analytics:
+    image: savantly/nexus-databoard:latest
+    container_name: savantly-custom-report
+    command: ["sh", "-c", "/entrypoint.sh"]
+    ports:
+      - 8501:8501
+    environment: []
+    volumes:
+      - ./pages:/app/pages
+      - ./entrypoint.sh:/entrypoint.sh
```

Comment:
This is trying to mount a `./pages` folder that doesn't exist

Reply:
i must be tired not get this. now will fix it